src/ucp/wireup/address.c

```diff
@@ -840,7 +841,7 @@ ucs_status_t ucp_address_unpack(ucp_worker_t *worker, const void *buffer,
         md_flags     = (md_byte & UCP_ADDRESS_FLAG_MD_ALLOC) ? UCT_MD_FLAG_ALLOC : 0;
         md_flags    |= (md_byte & UCP_ADDRESS_FLAG_MD_REG)   ? UCT_MD_FLAG_REG   : 0;
         empty_dev    = md_byte & UCP_ADDRESS_FLAG_EMPTY;
-        ptr          = UCS_PTR_TYPE_OFFSET(ptr, md_byte);
+        ptr          = UCS_PTR_TYPE_OFFSET(ptr, md_index);
```

Comment:
why need this change?

Reply:
when packing we adds `sizeof(md_index)` instead of `sizeof(md_byte)` - just to align
it fixes a potential bug if `md_index` type is changed

Reply:
we take `md_byte` from ptr address, it makes sense to advance pointer by its type, not `md_index`.
`md_index` is something we create based on `md_byte` value

Reply:
got it - thanks, removed